pom.xml

```diff
@@ -14,15 +14,15 @@
     <maven.compiler.target>8</maven.compiler.target>
     <maven.compiler.source>8</maven.compiler.source>
 
-    <io.jsonwebtoken.jjwt.version>0.11.2</io.jsonwebtoken.jjwt.version>
+    <io.jsonwebtoken.jjwt.version>0.12.3</io.jsonwebtoken.jjwt.version>
 
-    <junit.jupiter.version>5.8.2</junit.jupiter.version>
-    <assertj.core.version>3.22.0</assertj.core.version>
-    <mockito.core.version>4.3.1</mockito.core.version>
-    <findbugs.annotations.version>3.0.1</findbugs.annotations.version>
+    <junit.jupiter.version>5.10.1</junit.jupiter.version>
+    <assertj.core.version>3.24.2</assertj.core.version>
+    <mockito.core.version>4.11.0</mockito.core.version>
+    <findbugs.annotations.version>3.0.1u2</findbugs.annotations.version>
```

Comment:
According to [this](https://mvnrepository.com/artifact/com.google.code.findbugs/annotations), it looks like this was moved to [here](https://mvnrepository.com/artifact/com.github.spotbugs/spotbugs-annotations). That seems to also be in the same version as the existing spotbugs plugin that you're using now (4.8.2). It may be worth checking how to port this.